### PR TITLE
Feature/new homepage

### PR DESCRIPTION
**Pull request by cjstevens78**
Styling for the new homepage layout. Includes changes to:

- Search banner
- Box links
- Media with text default
- Media with text featured
- Button styling
- News list
- Global Footer
- Font Awesome Icons

### CODE REVIEW

File: assets/scss/lib/partials/_buttons.scss [hunk 1]
```diff
@@ -172,3 +172,56 @@
     }
   }
 }
+
+// special outlined links styled as buttons for the homepage
+
```
Comment (Adnan-cds) [inline assets/scss/lib/partials/_buttons.scss]: I don't think you want this gap :)

File: assets/scss/lib/partials/_shame.scss [hunk 2]
```diff
@@ -113,3 +113,11 @@ main a[href^="mailto:"],
 .node--type-localgov-subsites-page .layout--onecol__flexible .layout__region--content.row {
   margin-left: 0;
 }
+
+// flip the font awesome phone icon back the right way after font awesome changed its orientation iin the codebase
+
```
Comment (Adnan-cds) [inline assets/scss/lib/partials/_shame.scss]: Again, possibly an unwanted gap.  I can also see a typo in the preceding sentence.

File: assets/scss/lib/partials/_shame.scss [hunk 3]
```diff
@@ -113,3 +113,11 @@ main a[href^="mailto:"],
...
+.fa-phone {
+  -ms-filter: "progid:DXImageTransform.Microsoft.BasicImage(rotation=0, mirror=1)";
```
Comment (Adnan-cds) [inline assets/scss/lib/partials/_shame.scss]: Do we really need these -ms-filter's?  I thought the Microsoft Edge browser uses the Blink render engine which is the same as the Google Chrome.

Comment (cjstevens78) [reply assets/scss/lib/partials/_shame.scss]: I'll remove this - it was a cut and paste fix and will be removed soon when we properly go over to font awesome 6

Comment (Adnan-cds) [reply assets/scss/lib/partials/_shame.scss]: Thanks.

**Review by Adnan-cds — APPROVED** [review]
Thanks.  Looks okay now.

**Pull request merged**